custom_components/trakt_tv/sensor.py

```diff
@@ -151,3 +185,40 @@ def has_entity_name(self) -> bool:
     async def async_update(self):
         """Request coordinator to update data."""
         await self.coordinator.async_request_refresh()
+
+
+class TraktStateSensor(Entity):
+    """Trakt sensor to show data as state"""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        hass,
+        coordinator,
+        prefix: str,
+        mdi_icon: str,
+        title: str,
+        state: str,
+    ):
+        """Initialize the sensor."""
+        self.hass = hass
+        self.coordinator = coordinator
+        self.prefix = prefix
+        self.icon = mdi_icon
+        self.title = title
+        self.state = state
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self.prefix} {self.title}"
+
+    @property
```

Comment:
You replace in title any "_", the spit doesn't make sense here. Should not be self.title.split(" ")[0] because in episodes_plays, the unit is episodes not plays, right ?

Reply:
My bad with the underscore, nice catch.
The names are all over the place so I'm not sure how to solve it
- **Episodes** Collected
- Episodes **Comments**
- Episodes **Minutes**
- Episodes **Plays**
- Episodes **Ratings**
- **Episodes** Watched
Maybe having no unit of measurement makes more sense?

Reply:
I like the idea of an unit, we have limited choice so we can create a function to handle all the cases such as:
```
units = ["ratings", "minutes", "comments","friends","followiers","following","movies","shows","seasons"]
for unit in units:
  if lower(self.title).contains(unit):
    return unit
return None
``` (Not tested)

I know it is not straightforward but I think users don't care how it is implemented and just want it to work ^^'

Reply:
Thanks, this probably is the best way for it to work properly. Fixed now!

![image](https://github.com/dylandoamaral/trakt-integration/assets/70920705/54ce3063-c6a9-44ce-86b6-5b6df3cf0482)